### PR TITLE
Add email form

### LINKED ISSUE #18: [FEAT] - HTML Form for mailing list signup
### What feature would you like to see?

HTML form for mailing list signup > airtable api (data backend) > sendy

### Additional information

Going HTML form route to match branding on site as best as possible.

### PR DESCRIPTION
**Pull request by kcoderhtml**
# Add email form
## Description:

Add the form and a Netlify function to proxy the request.

## Related Issue:
Closes #18

### CODE REVIEW

File: public/bg.png
Comment (jdogcoder) [inline public/bg.png:1]: What's going on with this change @kcoderhtml?

CleanShot 2023-12-17 at 20 22 44@2x

Comment (kcoderhtml) [reply public/bg.png:1]: @CaenJones wanted to change it so that it wasn't overlapping and looking so much like an image.

Comment (kcoderhtml) [reply public/bg.png:1]: we may want to have @polypixeldev look at it and see if he has any thoughts.

Comment (jdogcoder) [reply public/bg.png:1]: I do like them both.

Comment (jdogcoder) [reply public/bg.png:1]: Let's get @polypixeldev on this tho :)

**Comment by polypixeldev** [comment]
goofy bug - bg image doesn't show by default but it does when i open devtools? firefox/linux
image

File: public/bg.png
Comment (polypixeldev) [reply public/bg.png:1]: looks good to me!

**Review by jdogcoder — APPROVED** [review]
LGTM

**Comment by jdogcoder** [comment]
Oh wait

**Review by jdogcoder — CHANGES_REQUESTED** [review]
can we fix the above first

**Comment by kcoderhtml** [comment]
@polypixeldev are you having the bg issue on the main site? https://purplebubble.org

**Comment by kcoderhtml** [comment]
I just tested firefox, chrome, and safari on mac and they all worked fine.

**Comment by kcoderhtml** [comment]
On a pixel 4 with chrome, it looks like below, which is a problem.  
image

**Comment by polypixeldev** [comment]
> @polypixeldev are you having the bg issue on the main site? https://purplebubble.org

I am getting the same issue on main site

**Comment by kcoderhtml** [comment]
weird; maybe its a config issue on your end? have you tried a private session or new profile?

**Review by jdogcoder — APPROVED** [review]

**Pull request merged**